lib/utils/createConfig.js

```diff
@@ -75,25 +70,11 @@ function createConfig(config, argv, { port }) {
     }
   }
 
-  if (argv.contentBase) {
-    options.contentBase = argv.contentBase;
-
-    if (Array.isArray(options.contentBase)) {
-      options.contentBase = options.contentBase.map((p) => path.resolve(p));
-    } else if (/^[0-9]$/.test(options.contentBase)) {
-      options.contentBase = +options.contentBase;
-    } else if (!isAbsoluteUrl(String(options.contentBase))) {
-      options.contentBase = path.resolve(options.contentBase);
-    }
-  }
-  // It is possible to disable the contentBase by using
-  // `--no-content-base`, which results in arg["content-base"] = false
-  else if (argv.contentBase === false) {
-    options.contentBase = false;
-  }
-
-  if (argv.watchContentBase) {
-    options.watchContentBase = true;
+  if (argv.static === false) {
+    options.static = false;
+  } else if (argv.static) {
+    options.static = argv.static.split(',');
+    options.static = options.static.map((staticDir) => path.resolve(staticDir));
```

Comment:
Added multiple handling here

Reply:
I think we should avoid this syntax, in favor multiple `--static=one --static=two`